### ISSUE TITLE
BaseSystemFields does not satisfy type Record of of Pocketbase JS SDK

### ISSUE THREAD

**Issue by Sadragos** [patmood/pocketbase-typegen#47]
I'm using the JS Pocketbase SDK to communicate with backend. It'd be great if the generated types would satisfy the Record type of said SDK so that the IDE does not throw this error when loading Files with the getFileUrl method.
```
Argument of type 'UsersResponse' is not assignable to parameter of type 'Record'.
  Type 'UsersResponse' is missing the following properties from type 'Record': load, loadExpand, isNew, clone, export
```

**Comment by patmood** [comment]
Thanks for flagging this! I haven’t used the file api so I’ll take a look and update the types

On Tue, Feb 14, 2023, at 2:01 AM, Sadragos wrote:
> 
> 
> I'm using the JS Pocketbase SDK to communicate with backend. It'd be great if the generated types would satify the Record type of said SDK so that the IDE does not throw this error when loading Files with the getFileUrl method.
> 
> `Argument of type 'UsersResponse' is not assignable to parameter of type 'Record'.
>   Type 'UsersResponse' is missing the following properties from type 'Record': load, loadExpand, isNew, clone, export
`

**Comment by patmood** [comment]
This is fixed in the [v0.10.2 release](https://github.com/pocketbase/js-sdk/releases/tag/v0.10.2) of the pocketbase SDK 🙌
